boards/f4vi1/include/periph_conf.h

```diff
@@ -78,28 +78,29 @@ static const timer_conf_t timer_config[] = {
 /** @} */
 
 /**
- * @name UART configuration
+ * @brief   UART configuration
```

Comment:
Also, check if this parses still correctly. Without the name the sectioning in the doc often isn't correct anymore (and the brief of the group is added to the first element)